views/partials/header.ejs

```diff
@@ -1,5 +1,12 @@
-<nav>
-    <a href="/">
-        <img src="/images/logo-desktop.png" alt=""> 
-    </a>
-</nav>
\ No newline at end of file
+<section>
+    <nav>
+        <a href="/">
+            <img src="/images/logo-desktop.png" alt=""> 
+        </a>
+        <ul>
+            <li><a href="/profiel">Mijn Profiel</a></li>
```

Comment:
Het is logischer om "Profiel" en "Likes" zonder hoofdletter te doen. verder ziet het er goed uit!

Reply:
jess ik ga ernaar kijken